jme3-core/src/main/java/com/jme3/util/mikktspace/MikktspaceTangentGenerator.java

```diff
@@ -279,7 +279,7 @@ public static boolean genTangSpace(MikkTSpaceContext mikkTSpace, final float ang
                 float tang[] = {pTSpace.os.x, pTSpace.os.y, pTSpace.os.z};
                 float bitang[] = {pTSpace.ot.x, pTSpace.ot.y, pTSpace.ot.z};
                 mikkTSpace.setTSpace(tang, bitang, pTSpace.magS, pTSpace.magT, pTSpace.orient, f, i);
-                mikkTSpace.setTSpaceBasic(tang, pTSpace.orient == true ? 1.0f : (-1.0f), f, i);
+                mikkTSpace.setTSpaceBasic(tang, pTSpace.orient == true ? -1.0f : 1.0f, f, i);
```

Comment:
Yes, but i've decided to not patch it, since it has other issues and it is there only to maintain legacy code.
Also, I am not sure from where that implementation comes from, so the "flipped" sign might be intended...

I would support adding a `@Deprecated` annotation to the old generator, but this should probably be addressed somewhere else.